actionpack/lib/action_dispatch/middleware/show_exceptions.rb

```diff
@@ -50,6 +50,8 @@ def render_exception(request, exception)
         request.path_info = "/#{status}"
         response = @exceptions_app.call(request.env)
         response[1]["X-Cascade"] == "pass" ? pass_response(status) : response
+      rescue Mime::Type::InvalidMimeType => e
```

Comment:
This middleware should not handle any special exceptions. All exceptions should be handled in the layer below. This middleware should only have the `rescue` in the line 55 since handing this here will not allow users to override the behavior to handle the exception with a different response code.

Reply:
I see what you mean. Techically they'll still be able to override this error if they rescue it in their exception app, but I agree that rescuing special error here seems weird.

We should probably add a note in the migrating guide to let people know they have to deal with this exception. Gonna close this and rescue it in our app

Reply:
:+1: for the note in the upgrade guide.